README.md

```diff
@@ -1,85 +1,135 @@
 # tar_scm (OBS source service) [![Build Status](https://travis-ci.org/openSUSE/obs-service-tar_scm.png?branch=master)](https://travis-ci.org/openSUSE/obs-service-tar_scm)
 
-This is an [Open Build Service](http://openbuildservice.org/) source
-service. It uses an SCM client to checkout or update a package by
-creating a source tarball from a source code repository.
+This is the git repository for
+[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm),
+which provides several [source
+services](http://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html)
+for the [Open Build Service](http://openbuildservice.org/) which all
+assist with packaging source code from SCM (source code management)
+repositories into tarballs.  The authoritative source is
+https://github.com/openSUSE/obs-service-tar_scm.
+
+## Services
+
+### tar_scm *(deprecated)*
+
+`tar_scm` is the legacy source service used to create a source tarball
+from one of the supported SCM (source code management) tools: `git`,
+`hg`, `svn`, and `bzr`.
 
-It supports many options, e.g. it can adjust resulting tarball
-parameters, include or exclude particular files when creating this
-tarball or generate an rpm changelog from the SCM commit log. For the
-full list of options please see tar_scm.service.in.
+`tar_scm` supports many options, e.g. it can adjust resulting tarball
+parameters, include or exclude particular files when creating the
+tarball, or generate an `rpm` changelog from the SCM commit log. For the
+full list of options please see `tar_scm.service.in`.
 
-Apart from various SCM like git, hg, bzr or svn it additionally
+Apart from various SCM like git, hg, bzr or svn, it additionally
 supports `--url` option that allows you to specify URL of the upstream
 tarball to be downloaded.
 
-This is the git repository for
-[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm).
-The authoritative source is
-https://github.com/openSUSE/obs-service-tar_scm
-
-The service can be used in combination with other services like
+`tar_scm` can be used in combination with other services like
 [download_files](https://github.com/openSUSE/obs-service-download_files),
 [extract_file](https://github.com/openSUSE/obs-service-extract_file),
```

Comment:
this one is actually not needed anymore here, because we have the "extract" parameter now. It only makes sense when you used download_url or download_files before. But IMHO out of scope here

Reply:
True - happy to drop this but I think a central index would be beneficial.

Reply:
Dropping this in next push ...